cmake/FindPython/Support.cmake

```diff
@@ -17,7 +17,7 @@ if (NOT DEFINED _${_PYTHON_PREFIX}_REQUIRED_VERSION_MAJOR)
   message (FATAL_ERROR "FindPython: INTERNAL ERROR")
 endif()
 if (_${_PYTHON_PREFIX}_REQUIRED_VERSION_MAJOR EQUAL 3)
-  set(_${_PYTHON_PREFIX}_VERSIONS 3.7 3.6 3.5 3.4 3.3 3.2 3.1 3.0)
+  set(_${_PYTHON_PREFIX}_VERSIONS 3.8 3.7 3.6 3.5 3.4 3.3 3.2 3.1 3.0)
```

Comment:
is this actually used anywhere? Things have been working fine with 3.8 for a while now...

Reply:
*fine....on windows at least.

Reply:
> is this actually used anywhere? Things have been working fine with 3.8 for a while now...

Really? I have Python 2.7 (system version) and Python 3.8 (from homebrew) and libCellML was always picking up Python 2.7. With that change, it now picks up Python 3.8.

Reply:
This is required when the FindPython script can find multiple Pythons.  If multiple Pythons are available then these versions are used to test for other Pythons.  This means that Python 3.8 cannot be found unless you add it as an option here.